audio/audacity/Portfile

```diff
@@ -81,34 +81,42 @@ if {${subport} eq "${name}-legacy"} {
     # get the source tarball from github because it contains all required external libs
     # incl. those not in MacPorts.
     github.setup    audacity audacity 2.4.2 Audacity-
-    revision        2
+    revision        4
```

Comment:
seems to be no reason to increase the revision to "4" - this should just be "3" here and below